ex7/index.js

```diff
@@ -0,0 +1,19 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert the number of apples purchased:");
+const apples = Number(prompt(">"));
+
+function shop() {
+    if (apples < 12) {
+        let applesLess = (apples * 0.30);
+        applesLess = applesLess.toFixed(2);
```

Comment:
The variables applesLess and applesMore are created just to store the total price, and then .toFixed(2) is used to format the number. Instead of creating separate variables and reasigning them, this can be simplified by formatting the result directly when creating the variable.

```suggestion
        const finalPrice = (apples * 0.30).toFixed(2);
```